src/components/Navbar.js

```diff
@@ -0,0 +1,22 @@
+import { Link } from 'react-router-dom';
+import React from 'react';
+
+function Navbar() {
+  return (
+    <header>
+      <nav className="d-flex navigation">
+        <ul className="d-flex">
+          <li><h1>Bookstore</h1></li>
+          <li>
+            <Link to="/">Books</Link>
+          </li>
+          <li>
+            <Link to="/authors">Authors</Link>
```

Comment:
- [ ] This should be `/categories` as stated in the project requirement. Like so 👇🏾 

```javascript
<Link to="/categories">Categories</Link>
```